packages/valory/skills/generic_scoring_abci/tests/test_payloads.py

```diff
@@ -54,6 +61,12 @@ def test_payloads(test_case: PayloadTestCase) -> None:
         sender="sender",
         content=test_case.content,
     )
+
     assert payload.sender == "sender"
     assert payload.content == test_case.content
-    assert payload.from_json(payload.json) == payload
+
+    if test_case.name == "Payload incorrect data type":
```

Comment:
Maybe move this into a separate test? It doesnt feel right to check for the test name, although I can see it works.

Reply:
Or just replace it with a flag, e.g., `raises` instead of an `str` name.